docs/nightly/en/user-guide/ingest-data/for-iot/grpc-sdks/template.md

```diff
@@ -51,57 +46,55 @@ To improve the efficiency of writing data, you can create multiple rows at once
 
 {template create-rows%%}
 
-#### Insert data
+### Insert data
 
 The following example shows how to insert rows to tables in GreptimeDB.
 
 {template insert-rows%%}
 
-#### Streaming insert
+### Streaming insert
 
 Streaming insert is useful when you want to insert a large amount of data such as importing historical data.
 
 {template streaming-insert%%}
 
 {template update-rows%%}
 
-<!-- TODO ### Delete Metrics -->
-
-### High-level API
+## High-level API
 
 The high-level API uses an ORM style object to write data to GreptimeDB.
 It allows you to create, insert, and update data in a more object-oriented way,
 providing developers with a friendlier experience.
 However, it is not as efficient as the low-level API.
 This is because the ORM style object may consume more resources and time when converting the objects.
 
-#### Create row objects
+### Create row objects
 
 {template high-level-style-object%%}
 
-#### Insert data
+### Insert data
 
 {template high-level-style-insert-data%%}
 
-#### Streaming insert
+### Streaming insert
 
 Streaming insert is useful when you want to insert a large amount of data such as importing historical data.
 
 {template high-level-style-streaming-insert%%}
 
 {template high-level-style-update-data%%}
 
-### More examples
+## More examples
 
 {template more-ingestion-examples%%}
 
 {template ingester-lib-debug-logs%%}
 
-### Ingester library reference
+## Ingester library reference
 
 {template ingester-lib-reference%%}
 
-## Query data
+<!-- ## Query data
```

Comment:
Adding a link to `Query data` document would be good.

Reply:
This section aims to introduce how to use other SDKs to query data, it will be moved to the SDK tool document later in the other refactor PRs.

Reply:
the query data links are at `/ingest-data/overview.md#next-steps`